calicoctl/calico_ctl/node.py

```diff
@@ -847,8 +848,13 @@ def handle_sigterm(sig, frame):
     except KeyboardInterrupt:
         # Mainline. Someone pressed Ctrl-C.
         print "Stopping Calico node..."
+        clean_exit = True
     finally:
         # Could either be this process is being killed, or output generator
         # raises an exception.
         if stop_container_on_exit:
             docker_client.stop(container)
+            # If the container is stopped due to unknown circumstances, exit
+            # with failed status code.
+            if not clean_exit:
+                sys.exit(1)
```

Comment:
Yes, I meant the `wait()` returns the status of the container, not the command.

And yes, we're in agreement about not really caring about the container status output, but it could be simpler if we have wait instead of attach to detect any error, rather than scrubbing for the string output, but I guess the string detection is already working so why fix it..